src/main/kotlin/com/sourcegraph/cody/internals/IgnoreOverrideAction.kt

```diff
@@ -62,6 +62,7 @@ class IgnoreOverrideDialog(val project: Project) : DialogWrapper(project) {
   }
 
   override fun doOKAction() {
+    super.doOKAction()
```

Comment:
it was a bug, the values were always from before the change. rather small impact tho as we use it only for our testing.